examples/graphiql-cdn/index.html

```diff
@@ -6,8 +6,9 @@
  *  LICENSE file in the root directory of this source tree.
 -->
 <!DOCTYPE html>
-<html>
+<html lang="en">
   <head>
+    <title></title>
```

Comment:
Can we add a proper title here?
```suggestion
    <title>GraphiQL</title>
```